src/reanimated2/InstallFunctions.js

```diff
@@ -87,12 +96,63 @@ export function installFunctions(innerNativeModule) {
     }
     _log(stringRepresentation(data))
   })
+
+  // clamp
+  const clamp = function (x, values) {
```

Comment:
by the wah, `function(x, ...values)` works just fine

Reply:
well, it's not :) some problems with Hermes which doesn't like the spread to be used at all

But works just fine on JSC